apps/cowswap-frontend/src/modules/ordersTable/pure/OrdersTableContainer/TableGroup.tsx

```diff
@@ -28,18 +30,83 @@ const Pagination = styled(OrdersTablePagination)`
   padding: 10px 0;
 `
 
+function TwapStatusAndToggle({
+  parent,
+  childrenLength,
+  isCollapsed,
+  onToggle,
+  onClick,
+  children,
+}: {
+  parent: any
+  childrenLength: number
+  isCollapsed: boolean
+  onToggle: () => void
+  onClick: () => void
+  children: any[]
```

Comment:
We don't use `any`

Reply:
`children` is used by React as a content for "generic box", we should not use it like here.
https://stackoverflow.com/questions/49706823/what-is-this-props-children-and-when-you-should-use-it